Exercises/arrays/arrays.js

```diff
@@ -0,0 +1,35 @@
+//Make a function which will generate array with random length and random values.
+
+// eslint-disable-next-line array-callback-return
+function randomArray() {
+  return Array.from(new Array(Math.floor(Math.random() * 100)), () => { return Math.floor(Math.random() * 10); });
+}
```

Comment:
You can remove code block and `return` keyword in your callback function. Probably then you wont need to disable eslint rule.